server/resolvers/usersResolver.js

```diff
@@ -1,9 +1,12 @@
-const { getUsers } = require('../models/services.deprecated/UserService');
+const { getUsers } = require('../models/services/UserService');
+
+const usersResolver = (_, __, context) => {
+    const { transaction } = context;
 
-const usersResolver = () => {
     // No authentication since participation is public!
-    return getUsers(null, {}, null, null, null, null, {
-        order: [['username', 'ASC']]
+    return getUsers({
```

Comment:
Seems `userAttributes`, `roleAttributes`, `atAttributes` and `testPlanRunAttributes` should be set to `[]` to maintain the current behavior

Reply:
Thanks for noting that, I set them to null which should have exactly the same effect as now.